lib/logstash/inputs/http.rb

```diff
@@ -243,19 +238,19 @@ def build_ssl_params
   end
 
   def ssl_key_configured?
-    !!(@ssl_certificate && @ssl_key)
+    !(@ssl_certificate && @ssl_key).nil?
```

Comment:
let's skip these changes, IMO it's debatable which one of `!!expr` or `!(expr).nil?` is better